ibtk/src/lagrangian/FEProjector.cpp

```diff
@@ -665,12 +666,19 @@ FEProjector::computeL2Projection(PetscVector<double>& U_vec,
         IBTK_CHKERRQ(ierr);
         ierr = KSPSetFromOptions(solver->ksp());
         IBTK_CHKERRQ(ierr);
+
+        auto pair = d_initial_guesses.emplace(system_name, d_num_fischer_vectors);
+        FischerGuess& fischer_guess = (pair.first)->second;
+
+        fischer_guess.guess(U_vec, F_vec);
```

Comment:
Should the documentation make it clear this is only done for "regular" L2 projections?

I guess the modified projection is not yet in the main branch...

Reply:
Also --- should we bypass if the number of vectors is 0?

Reply:
We do - see https://github.com/drwells/IBAMR/blob/Fischer/ibtk/src/lagrangian/FischerGuess.cpp#L102